volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -131,18 +154,18 @@ def enumerate_mft_records(
                     # Update the base offset to point to the next attribute
                     attr_base_offset += attr.Attr_Header.Length
                     # Get the next attribute
-                    attr = context.object(
-                        attribute_object,
+                    attr: mft.MFTAttribute = context.object(
+                        attribute_object_typ_name,
                         offset=offset + attr_base_offset,
                         layer_name=layer.name,
                     )
 
     @classmethod
     def parse_mft_records(
         cls,
-        record_map: Dict[int, Tuple[str, int, int]],
-        mft_record: interfaces.objects.ObjectInterface,
-        attr: interfaces.objects.ObjectInterface,
+        record_map: DefaultDict[str, MFTRecord],
```

Comment:
```suggestion
        record_map: Dict[str, MFTRecord],
```
The type here could surely just be a Dict?